src/app/patient-dashboard/common/locator-map/locator-map.component.ts

```diff
@@ -1,15 +1,17 @@
 
 import { take } from 'rxjs/operators';
-import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Component, OnInit, OnDestroy, Input } from '@angular/core';
 import { Observable, Subject, Subscription } from 'rxjs';
 import { flatMap } from 'rxjs/operators';
 
+
 import * as _ from 'lodash';
 
 import { FileUploadResourceService } from '../../../etl-api/file-upload-resource.service';
 import { PersonResourceService } from '../../../openmrs-api/person-resource.service';
 import { AppSettingsService } from '../../../app-settings/app-settings.service';
 import { PatientService } from '../../services/patient.service';
+import { DataSource } from '@angular/cdk/table';
 
```

Comment:
3rd party and angular imports usually go to the top of the file, before app imports. So it's usually
angular imports
..
3rd part imports
...
ampath and app imports